.github/ISSUE_TEMPLATE/Plugin_Issue.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Plugin Request
+about: Create a plugin request to discuss adding a new plugin to Streamlink
+
+---
+
+<!--
+Thanks for requesting a plugin!
+
+Please ensure that your request meets the requirements (https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#plugin-requests) and does not fall within the categories we will not implement. Plugin requests which do not meet these requirements will be closed.
+
+Please read the contribution guidelines (https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink) first!
+
+Also check the list of open and closed plugin requests before making a new request! (https://github.com/streamlink/streamlink/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22plugin+request%22+)
+
+Please fill out the following template. Be as detailed as possible.
+
+Please see the text preview to avoid unnecessary formatting errors.
+-->
+
+----
+
+## Plugin Issue
+
+- [ ] This is a plugin issue and I have read the contribution guidelines. <!-- Replace the space with an x to check the box: [x] -->
+
+### Description
+
+<!-- Explain the plugin issue as clearly as you can, which plugin is encountering isuses, what the issue is, etc. -->
+
+...
+
+### Expected / Actual behavior
```

Comment:
Do we need this section here? The description section should be enough, I think.